xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearch.xml

```diff
@@ -250,18 +258,19 @@
       window.history.replaceState &amp;&amp; window.history.replaceState(state, document.title);
       // Make sure the browser address bar reflects the new state (and thus the new state can be bookmarked).
       window.history.pushState &amp;&amp; window.history.pushState(state, document.title, viewURL);
+      document.querySelector(changeTargetSelector).focus();
```

Comment:
```suggestion
      document.querySelector(changeTargetSelector)?.focus();
```

Just to be safe, in case the selector is not matched after the reload, for some unforeseen reason.